app/controllers/AdminController.php

```diff
@@ -345,9 +345,22 @@ public function deleteRole(){
 
 	public function credentialStoreView(){
         Session::put("admin-nav", "credential-store");
-        $tokens = AdminUtilities::get_all_ssh_tokens();
+        $tokens = AdminUtilities::get_all_ssh_tokens_with_description();
+        //Auto-generating the Default SSH Key if there is no key with such description.
```

Comment:
We don't need to generate a "Default SSH Key".

Reply:
Hi Marcus,

The reason why I'm doing this is if this code is pushed live, most of the people might not have key with description "default SSH Key". As this key is supposed to be default, I'm creating one for them. If a default key is not necessary, I'll remove this!

Thanks

Reply:
Yeah, let's just not do anything special with credentials based on their description.  Existing gateways will already have a "default" SSH key.